pytition/locale/fr_FR/LC_MESSAGES/django.po

```diff
@@ -1275,6 +1282,82 @@ msgstr "Voulez-vous vraiment faire cela ?"
 msgid "Close"
 msgstr "Fermer"
 
+#: pytition/petition/templates/petition/transfer_petition.html:9
+#, fuzzy
+#| msgid "Create a petition"
+msgid "Transfer petition"
+msgstr "Créer une pétition"
+
+#: pytition/petition/templates/petition/transfer_petition.html:15
+#, fuzzy
+#| msgid ""
+#| "\n"
+#| "            Do you really want to delete petition '%(title)s'\n"
+#| "          "
+msgid ""
+"\n"
+"              You are going to transfer this petition to someone else:\n"
+"              "
+msgstr ""
+"\n"
+"Voulez-vous vraiment supprimer la pétition  '%(title)s'\n"
+"          "
```

Comment:
There was probably some mix-up with this translation ;)
Same applies to the following ones, I'll let you take a look

Reply:
Ah, right! Thanks!